packages/wallet/src/services/types.ts

```diff
@@ -98,7 +99,19 @@ export interface RewardsHistory {
   lifetimeRewards: Cardano.Lovelace;
 }
 
+/** Wallet delegated stake by pool. Multiple reward accounts could be delegated to the same pool */
+export interface StakeByPool {
+  pool: Cardano.StakePool;
+  /** How much from the total delegated stake is delegated to this pool */
+  percentage: Percent;
+  /** Absolute stake value */
+  value: bigint;
```

Comment:
`Value` has a specific meaning - structure that has both coins and asset
```suggestion
  stake: bigint;
```

Reply:
Updated here: [d4e79ed](https://github.com/input-output-hk/cardano-js-sdk/pull/735/commits/d4e79ed6e4d7cc95b1ceaeb19ad830bfe2c058c1)